### ISSUE TITLE
Select hardware inventory tracking solution

### ISSUE THREAD

**Issue by larsks** [CCI-MOC/ops-issues#6]
We need to select an inventory management solution:

- Which platform? Augustine is using Netbox and says it's simple.
- Self hosted or pay for hosting?

**Comment by larsks** [comment]
If we want to look at Netbox it looks like there is a community maintained docker image at https://hub.docker.com/r/netboxcommunity/netbox

**Comment by larsks** [comment]
Interesting article on using Netbox as the source for network device configuration here: https://blogs.cisco.com/developer/source-of-truth-cleur

**Comment by larsks** [comment]
I've set up a netbox instance in openshift at https://netbox-moc-ops.apps.cnv.massopen.cloud/ and populated it with a few devices from the openshift and openstack clusters. The password for the `admin` user is in bitwarden.

**Comment by msdisme** [comment]
Cool- thanks!

________________________________

**Comment by larsks** [comment]
Another option for asset tracking is [GLPI](https://glpi-project.org/). I've set up a GLPI demo (but I have not populated it with any data yet) at https://glpi-moc-ops.apps.cnv.massopen.cloud/. The password for the `glpi` user is in bitwarden.

**Comment by larsks** [comment]
[Ralph](https://ralph.allegro.tech/) looks interested and has a live demo (https://ralph-demo.allegro.tech/, login `ralph`/`ralph`).

**Comment by larsks** [comment]
...and as we mentioned on `#coredev` earlier today, [Snipe-IT](https://snipeitapp.com/) is open source and has both self-hosted and vendor-hosted options.

**Comment by msdisme** [comment]
I bumped this up the backlog so we consider it for inclusion in the next sprint.

**Comment by naved001** [comment]
Out of all the options I like netbox the best and would vote for that.

Other options I looked at:
1. Ralph 3: This came close to netbox as it had everything and is built on django/python but it didn't have mechanism to track cables between hosts.
2. snipeit: seemed more for asses tracking and office stuff.
3. opendcim: looks like it has most features, but the ui is too old and clunky.
4. glpi: same as opendcim. I think @msd and I did spend some creating stuff on it together few months ago.
